test/test_image_processing_utils.py

```diff
@@ -0,0 +1,193 @@
+import nway.image_processing_utils as imu
+import pytest
+from pathlib import Path
+import numpy as np
+import PIL.Image
+import cv2
+
+TEST_FILE_DIR = Path.resolve(Path(__file__)).parent / "test_files"
+
+
+@pytest.fixture
+def fixed_image():
+    # grab one specific file because failing large translation
+    # cases vary image-by-image
+    fgen = TEST_FILE_DIR.rglob("avgInt_a1X.png")
+    for f in fgen:
+        if "ophys_session_775289198" in str(f):
+            impath = f
+    with PIL.Image.open(impath) as fp:
+        image = np.array(fp)
+    return image
+
+
+@pytest.fixture
+def noisy_image(fixed_image):
+    flatim = np.flipud(np.fliplr(fixed_image))
+    return flatim
+
+
+@pytest.fixture
+def moving_image_fixture(fixed_image, request):
```

Comment:
For consistency either rename this to `moving_image` or rename `fixed_image` and `noisy_image` to `fixed_image_fixture` and `noisy_image_fixture`.

Reply:
done. I also saw that I had retained the `noisy` name when in fact I found that flipping was better for causing a failure to register. renamed accordingly.